### PR TITLE
Feature/tao 6924 move to

### PR DESCRIPTION
**Pull request by jsconan**
Related to: https://oat-sa.atlassian.net/browse/TAO-6924

Implement the `moveTo` action

### CODE REVIEW

File: views/js/layout/actions/common.js [hunk 1]
```diff
@@ -523,6 +533,130 @@ define([
                 .on('error', reject);
             });
         });
+
+        /**
+         * Register the moveAll action: select a destination class to move resources
```
Comment (krampstudio) [inline views/js/layout/actions/common.js]: The action is called `moveTo`

File: views/js/layout/actions/common.js [hunk 2]
```diff
@@ -523,6 +533,130 @@ define([
...
+         *
+         * @this the action (once register it is bound to an action object)
+         *
+         * @param {Object|Object[]} actionContext - multiple action contexts
+         * @returns {Promise<String>} with the new resource URI
```
Comment (krampstudio) [inline views/js/layout/actions/common.js]: It resolve with the destination class URI

**Comment by krampstudio** [comment]
There's also 2 points, but to confirm with PM : 
 -  in the previous move implementation (one resource) you were warned about a potential lost of metadata only when the properties of the destination did not match the properties from the copies (there was a method server side to compute the property diff)
 - the warning message may need to be reviewed. I find it understandable because I know the underlying concepts, but for an end user, this message can be confusing. I don't know if we can find a more concise version.

**Comment by jsconan** [comment]
Indeed @krampstudio, the warning message was not so good. It is now changed to something clearer and more concise.

For the backend diff/confirm feature, it would be a little tricky to deal with an huge selection of resources to move. After having discussed that with PM, we agreed to keep the generic message approach.

**Pull request merged**